Sources/GoodExtensions/CG/CGFloat.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  File.swift
+//  
+//
+//  Created by Andrej Jasso on 24/01/2022.
+//
+
+import CoreGraphics
```

Comment:
Import asi nieje potrebny

Reply:
No CoreGraphic je mensi ako cely Foundation, akonahle som pouzil CGFloat sa sam importoval toto mi robi vzdy ak nemam aspon Foundation. Myslim ze to tam treba